shopcart.py

```diff
@@ -237,6 +237,16 @@ def create_products(sid):
         return make_response(jsonify(message), rc)
 
 ######################################################################
+# DELETE A SHOPPING CART
+######################################################################
+@app.route('/shopcarts/<int:sid>', methods=['DELETE'])
+def delete_shopcarts(sid):
+    for i in range(len(shopping_carts)):
+    	if shopping_carts[i]['sid'] == sid:
+    	    del shopping_carts[i]
+    	    return '', HTTP_204_NO_CONTENT
```

Comment:
Yes product gets deleted, if you repeat it shows a blank screen and HTTP_204_NO_CONTENT